tests/e2e_tests/multistep/test_incentive.py

```diff
@@ -163,11 +159,11 @@ async def test_incentive(local_chain):
 
     # Create tasks to read stdout and stderr concurrently and write output to log file
     # ignore, don't await coroutine, just write logs to file
-    await asyncio.create_task(
+    asyncio.create_task(
         write_output_log_to_file("validator_stdout", validator_process.stdout)
     )
     # ignore, dont await coroutine, just write logs to file
-    await asyncio.create_task(
+    asyncio.create_task(
```

Comment:
Need to be awaited.

Reply:
According to Daniel, he uses this for debugging, so he asked to ignore it.